modules/installation-operators-config.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-operators-config-{context}"]
+= Initial Operator configuration
+
+After the control plane initializes, you must immediately configure some
+Operators so that they all become available.
+
+.Prerequisites
+
+* Your control plane has initialized.
+
+.Procedure
+
+. Watch the cluster components come online:
```

Comment:
Is this the recommended approach being given for other UPI installations? I woud have expected us to tell the user to use `wait-for install-complete` first and only look at clusteroperators if there was a problem?

Reply:
Yes, you do. There are a couple of Operators that are expected to not come online at this point, like registry, and I needed a way to watch for them.